src/AEG-IO.ts

```diff
@@ -45,21 +46,21 @@ interface atomObj {
  */
 interface proofNodeObj {
     tree: sheetObj;
-    appliedRule: string;
+    appliedRule: ProofModeMove;
```

Comment:
This is a massive API break, does loading files work, how doe the Enums get stored and loaded from json?

Reply:
Discussed in the PMH server on 3/21